h2o-r/tests/testdir_misc/runit_make_leaderboard.R

```diff
@@ -0,0 +1,62 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues = TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+test_make_leaderboard <- function() {
```

Comment:
looks like multiple tests in one, can we split it? easier to identify the different scenarios being tested (different test name), better error reporting, easier to read, to verify, and to modify if needed